docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Installing-Loris-in-Depth.md

```diff
@@ -0,0 +1,195 @@
+instructions on installation.
+
+We also have instructions for how to set up LORIS for macOS. However, only
+ Ubuntu and CentOS are officially supported by the team so we may be unable to
+ help you with issues arising on other systems.
+
+
+When you've completed this guide, further set up is required. Please visit
+[the Setup page](https://github.com/aces/Loris/wiki/Setup) in order to be
+able to use LORIS properly.
+
+## Prequisities
+
+### Server Stack
+
+LORIS requires a LAMP stack in order to run, specifically:
+* Apache 2.4
+* MySQL 5.7 (or MariaDB 10.1) (or higher)
+* PHP 7.2 (or higher)
+
+Additionally, LORIS requires the following package managers:
+* npm 8.0 (or higher)
+* composer
+
+This guide does not cover installation of these requirements.
+
+### Apt Packages
+The following Ubuntu packages are required and should be installed using 
+`sudo apt install ...`.
+
+* git
+* curl
+* wget
+* zip
+* unzip
+* php-json
+* make
+* software-properties-common
+* php7.2-mysql
+* php7.2-xml
+* php7.2-json
+* php7.2-mbstring
+* php7.2-gd
+* libapache2-mod-php7.2
+
+### Database set up
+
+MySQL (or MariaDB) must be installed and a root or admin-level MySQL user must
+be created before continuing. (This is not the same as a unix root credential.)
+
+## Getting the source code
+
+### LORIS for Development
+If installing LORIS for development purposes, clone the source code from our
+GitHub repository to the web root. Then change to this directory.
+
+Example command assuming LORIS will be served from `/var/www/`:
+
+```bash
+git clone git@github.com:aces/Loris.git /var/www/loris
+cd /var/www/loris/
+```
+
+### LORIS for Production
+
+Visit the releases page and download the zipped file of the latest LORIS
+release. The below command will download the latest LORIS release and save 
+it to a file named `loris-src.tar.gz`.
+
+```bash
+curl -s https://api.github.com/repos/aces/loris/releases/latest \
+| grep 'tarball' \
+| cut -d : -f2,3 \
+| tr -d \" \
+| tr -d , \
+| wget -i - -O loris-src.tar.gz
+```
+
+When this is complete, expand the compressed file and move its contents to the
+directory from which LORIS will be served. 
+
+```bash
+tar -zxf loris-src.tar.gz
+```
+
+This will create a folder called something like `aces-Loris-9e30cf0`. (The
+part after `aces-Loris` is not important).
+
+Rename this file to `loris`, e.g.
+
+```bash
+mv aces-Loris-9e30cf0 loris
+```
+
+Then move this folder to the web root (e.g. `/var/www/`) and go to this 
+directory.
+
+```
+mv ./loris /var/www/loris/
+cd /var/www/loris/
+```
+
+## Running the install script
+
+The next step in setting up LORIS is running the script `install.sh` in the 
+`tools/` directory. The script must be run from this directory.
+
+This will begin an interactive setup process that will configure files and
+permissions required to get LORIS up and running.
+
+The install script will prompt for a `root` password in order to configure
```

Comment:
```suggestion
The install script will prompt for the user password in order to use `sudo` permission for configuring
```
A note specifying that the user should be a member of the `sudo`group would be helpfull here.